src/Deposit.js

```diff
@@ -1016,4 +1022,62 @@ export default class Deposit {
       digest: _digest
     }
   }
+
+  /**
+   * Get the current collateralization level for this Deposit
+   * @return {number} Percentage collateralisation, as an integer. eg. 120%
+   */
+  async getCollateralisation() {
```

Comment:
British spelling alert. Calling out because it's inconsistent

Reply:
Oops.